metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -318,6 +318,9 @@ private boolean isEnabledForTableDefinitionMetadataDelete(final QualifiedName ta
     @Override
     public Optional<TableDto> get(final QualifiedName name, final GetTableServiceParameters getTableServiceParameters) {
         validate(name);
+
+        logGetTableMetrics(name, getTableServiceParameters);
```

Comment:
We log request metrics in https://github.com/Netflix/metacat/blob/master/metacat-main/src/main/java/com/netflix/metacat/main/api/RequestWrapper.java. Should we add this logic there?

Reply:
Much better; thanks.